completion/kubie.zsh

```diff
@@ -0,0 +1,36 @@
+function _kubie {
```

Comment:
At least upon some quick checking, the auto-completions work form me! 

However, I needed to prefix the file with `#compdef kubie`. I think such a header comment is always needed (e.g., see [kubectl code](https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/completion/completion.go#L174) for `kubectl completion zsh`. Please incorporate the suggestion:

```suggestion
#compdef kubie

function _kubie {
```